### PR TITLE
Gonryu patch 1

### PR DESCRIPTION
**Pull request by gonryu**
I added guide template for each SDK. All SDK have different TOC from each other, so we need to give consistency among them.

### CODE REVIEW

File: icon-key-concepts/account.md [hunk 1]
```diff
@@ -2,8 +2,14 @@ Account Management
 ==============
 
 ## Account
+Accounts represent identities of participants in the blockchain network, that is, users and contracts. Accounts are used to identify the owner of trasactions, and specify the destinations of trasactions. All accounts can have tokens.
```
Comment (sojinkim-icon) [inline icon-key-concepts/account.md]:
```suggestion
Accounts represent identities of participants in the blockchain network, that is, users and contracts. Accounts are used to identify the owner of transactions and specify the destinations of transactions. All accounts can have an ICX balance.
```

File: icon-key-concepts/account.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
+
 There are two types of accounts in ICON, accounts that are associated with users, often dubbed as Externally Owned Accounts (EOA), and Smart Contract (SCORE) Accounts. An EOA address starts with "hx" followed by a 20-byte hex string, while a Smart Contract Account address starts with "cx" followed by a 20-byte hex string. 
 
-In order to deploy a SCORE on the ICON network or make a transaction, one must hold a valid EOA. Furthermore, when we say account in this document, it means EOA. The terms of EOA, wallet, and keystore are not exactly same, but are often used interchangeably. 
+EOA has an associated private key, can sign and send a transaction, therefore can be the owner of transactions. However, a contract address does not have a private key, and cannot initiate a transaction. A contract address can only be the destination of transactions.  
+
+There are some special, pre-defined accounts, such as:
```
Comment (sojinkim-icon) [inline icon-key-concepts/account.md:13]: would be more kind to explain the accounts' types, roles, characteristics

File: icon-key-concepts/account.md [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
+ - Governance contract
+ - Public Treasury
+
+In order to deploy a SCORE on the ICON network or make a transaction, one must hold a valid EOA. Furthermore, when we say account in this document, it means EOA. The terms of EOA, wallet, and keystore are not exactly the same, but are often used interchangeably. 
 
-An account is cryptographically defined by a private-public key pair, and the account address can be derived from its public key. Creating an account is equivalant to creating a key pair, and the account can be exported as a file known as a keystore file. 
+An account is cryptographically defined by a private-public key pair, and the account address can be derived from its public key. Creating an account is equivalent to creating a key pair, and the account can be exported as a file known as a keystore file.
```
Comment (sojinkim-icon) [inline icon-key-concepts/account.md]:
```suggestion
An account is cryptographically defined by a private-public key pair, and the account address can be derived from its public key by taking the last 20-bytes of the SHA3-256 hash of the public key. Creating an account is equivalent to creating a key pair, and the account can be exported as a file known as a keystore file. 
```

File: icon-key-concepts/icon-nodes.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+ICON Network
+==============
+
+This document presents what kinds of nodes are in the ICON network. 
+Node means the computer server which participates in the blockchain protocol.
+All nodes keep the full or partial copy of blockchain data, and excute transactions in the blocks for validation.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
All nodes keep the full or partial copy of blockchain data, and execute transactions in the blocks for validation.
```

File: icon-key-concepts/icon-nodes.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+ICON Network
+==============
+
+This document presents what kinds of nodes are in the ICON network. 
+Node means the computer server which participates in the blockchain protocol.
+All nodes keep the full or partial copy of blockchain data, and excute transactions in the blocks for validation.
+So all nodes should be connected to internet, and have their own address to identify themselves.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
So all nodes should be connected to the internet, and have their own address to identify themselves.
```

File: icon-key-concepts/icon-nodes.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+==============
+
+This document presents what kinds of nodes are in the ICON network. 
+Node means the computer server which participates in the blockchain protocol.
+All nodes keep the full or partial copy of blockchain data, and excute transactions in the blocks for validation.
+So all nodes should be connected to internet, and have their own address to identify themselves.
+
+## Types of Nodes
+
+There are three types of nodes in ICON netwrok.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
There are three types of nodes in ICON network.
```

File: icon-key-concepts/icon-nodes.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+Peer nodes can participate in the consensus protocl, so they can produce and propose block.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
**Peer nodes** can participate in the consensus protocol, so they can produce and propose a block.
```

File: icon-key-concepts/icon-nodes.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+Citizen nodes just sychronize the blockchain data, and relay the transaction to Peer node.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
**Citizen nodes** just synchronize the blockchain data and relay the transaction to the Peer node.
```

File: icon-key-concepts/icon-nodes.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+Light clients just synchronize the header of blocks for simple verification of transactions.
+
+![types of nodes](types_of_nodes.png)
+
+The user and smart contract can access (query or send trasactoins) these nodes through ICONex wallet or ICON SDK.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
The user and application can access (query or send transactions) these nodes through ICONex wallet or ICON SDK. 
```

File: icon-key-concepts/icon-nodes.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+There are three types of nodes in ICON netwrok.
+Peer nodes can participate in the consensus protocl, so they can produce and propose block.
+Citizen nodes just sychronize the blockchain data, and relay the transaction to Peer node.
+Light clients just synchronize the header of blocks for simple verification of transactions.
+
+![types of nodes](types_of_nodes.png)
+
+The user and smart contract can access (query or send trasactoins) these nodes through ICONex wallet or ICON SDK. 
+Of course, they can also access the nodes using JSON RPC directly.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
Indeed, they can also access the nodes using JSON RPC directly as well.
```

File: icon-key-concepts/icon-nodes.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+## Peer
+
+There are two type of Peer nodes, Public and Commuinity Representitive.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
There are two types of Peer nodes, Public and Community Representative. 
```

File: icon-key-concepts/icon-nodes.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+![types of nodes](types_of_nodes.png)
+
+The user and smart contract can access (query or send trasactoins) these nodes through ICONex wallet or ICON SDK. 
+Of course, they can also access the nodes using JSON RPC directly.
+
+## Peer
+
+There are two type of Peer nodes, Public and Commuinity Representitive. 
+As you see in the word "Representitive", these nodes are supposed to be elected by ICONist or Community members.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
As you see in the word "Representative", these nodes are supposed to be elected by ICONist or Community members.
```

File: icon-key-concepts/icon-nodes.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+Node means the computer server which participates in the blockchain protocol.
+All nodes keep the full or partial copy of blockchain data, and excute transactions in the blocks for validation.
+So all nodes should be connected to internet, and have their own address to identify themselves.
+
+## Types of Nodes
+
+There are three types of nodes in ICON netwrok.
+Peer nodes can participate in the consensus protocl, so they can produce and propose block.
+Citizen nodes just sychronize the blockchain data, and relay the transaction to Peer node.
+Light clients just synchronize the header of blocks for simple verification of transactions.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
**Light clients** just synchronize the header of blocks for simple verification of transactions.
```

File: icon-key-concepts/icon-nodes.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+## Peer
+
+There are two type of Peer nodes, Public and Commuinity Representitive. 
+As you see in the word "Representitive", these nodes are supposed to be elected by ICONist or Community members.
+Details of election are presented in [ICONSENSUS](https://icon.community/iconsensus/).
+
+Peer nodes are the essential entities of ICON network. 
+They have roles to produce and validate the blocks, which contains transactions transmitted into ICON network.
+All transactions sent to ICON network are relayed or directed to Peer nodes.
+One of Peer nodes becomes the leader node, who have right to produce the block for his turn.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
One of Peer nodes becomes the leader node, who has the right to propose a block for its turn. 
```

File: icon-key-concepts/icon-nodes.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+There are two type of Peer nodes, Public and Commuinity Representitive. 
+As you see in the word "Representitive", these nodes are supposed to be elected by ICONist or Community members.
+Details of election are presented in [ICONSENSUS](https://icon.community/iconsensus/).
+
+Peer nodes are the essential entities of ICON network. 
+They have roles to produce and validate the blocks, which contains transactions transmitted into ICON network.
+All transactions sent to ICON network are relayed or directed to Peer nodes.
+One of Peer nodes becomes the leader node, who have right to produce the block for his turn. 
+And the other Peer nodes validate that block. The block is confirmed when 2/3 of Peer nodes agree on that block.
+Peer nodes will become a leader node in pre-defined order, and produce one block for their turn.
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
Peer nodes will become a leader node in a pre-defined order, and produce one block for their turn.
```

File: icon-key-concepts/icon-nodes.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+
+### Public Representitive (P-Rep)
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
### Public Representative (P-Rep)
```

File: icon-key-concepts/icon-nodes.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+
+### Community Represesntitive (C-Rep)
```
Comment (sojinkim-icon) [inline icon-key-concepts/icon-nodes.md]:
```suggestion
### Community Representative (C-Rep)
```

Comment (Spl3en) [reply icon-key-concepts/icon-nodes.md]: Representative

**Pull request merged**